src/content/peerconnection/trickle-ice/js/main.js

```diff
@@ -32,6 +41,35 @@ let begin;
 let pc;
 let candidates;
 
+const allServersKey = 'servers';
+
+function writeServersToLocalStorage() {
+  const serversSelect = document.querySelector('select#servers');
+  const allServers = JSON.stringify(Object.values(serversSelect.options).map(o => JSON.parse(o.value)));
+  window.localStorage.setItem(allServersKey, allServers);
+}
+
+function readServersFromLocalStorage() {
+  document.querySelectorAll('select#servers option').forEach(option => option.remove());
+  const serversSelect = document.querySelector('select#servers');
+  const storedServers = window.localStorage.getItem(allServersKey);
+
+  if (storedServers === null || storedServers === '') {
+    const o = document.createElement('option');
+    o.value = '{"urls":["stun:stun.l.google.com:19302"]}';
+    o.text = 'stun:stun.l.google.com:19302';
```

Comment:
You don't want all these specific values in two places. Make a function or a constant for "setDefaultServers" and call it from both places.